lib/xcodeproj/scheme/environment_variables.rb

```diff
@@ -0,0 +1,117 @@
+require 'xcodeproj/scheme/xml_element_wrapper'
+
+module Xcodeproj
+  class XCScheme
+    VARIABLES_NODE = 'EnvironmentVariables'
+    VARIABLE_NODE = 'EnvironmentVariable'
+
+    # This class wraps the EnvironmentVariables node of a .xcscheme XML file. This
+    # is just a container of EnvironmentVariable objects. It can either appear on a
+    # LaunchAction or TestAction scheme group.
+    #
+    class EnvironmentVariables < XMLElementWrapper
+      # @param [REXML::Element,Array{EnvironmentVariable},Array{Hash{String => String}}] node_or_target
+      #   The 'EnvironmentVariables' XML node that this object represents.
+      #   If nil, it will create a default XML node to use.
+      #
+      def initialize(node_or_target = nil)
```

Comment:
Change the name of the parameter from `node_or_target` to something more suitable like `node_or_variables` for example